addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -814,6 +814,44 @@ a.no-decoration {
     }
 }
 
-.accordion-button:not(.collapsed).bg-transparent::after {
-    background-image: escape-svg($accordion-button-icon);
-}
\ No newline at end of file
+.oe_website_sale {
+    // In website_sale the --accordion-active-bg is transparent or inherit
```

Comment:
Not sure to understand this comment 🤔
Are you pointing out an "unsolvable" bug, or does the following rule actually solve the issue?
I assumed it was the latter since I couldn't replicate the bug.
However, the CSS rule was already in place, so I'm a bit confused. 🤷🏻‍♂️😅